osbenchmark/workload/workload.py

```diff
@@ -746,6 +747,8 @@ def from_hyphenated_string(cls, v):
             return OperationType.RegisterMlModel
         elif v == "deploy-ml-model":
             return OperationType.DeployMlModel
+        elif v == "train-k-n-n-model":
```

Comment:
```suggestion
        elif v == "train-knn-model":
```

Reply:
I'll fix this by changing the name to TrainKnnModel. It seems like hyphens are automatically placed between every capitalized letter.